usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml

```diff
@@ -0,0 +1,39 @@
+name: SeaClouds Planner
+services:
+- type: brooklyn.entity.basic.VanillaSoftwareProcess
+  name: Planner Rest Service
+  brooklyn.config:
+    planner.config.port: 1234
+    planner.config.discoverer.url: undefined
+    planner.config.optimizer.url: undefined
+    planner.config.monitor_generator.url: undefined
+    planner.config.sla_generator.url: undefined
+    planner.config.service.url: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=planner-service&v=LATEST&e=jar
+
+  shell.env:
+    PLANNER_PORT: $brooklyn:config("planner.config.port")
+    DISCOVERER_URL: $brooklyn:config("planner.config.discoverer.url")
+    OPTIMIZER_URL: $brooklyn:config("planner.config.optimizer.url")
+    MONITOR_GENERATOR_URL: $brooklyn:config("planner.config.monitor_generator.url")
+    SLA_GENERATOR_URL: $brooklyn:config("planner.config.sla_generator.url")
+    PLANNER_SERVICE_URL: $brooklyn:config("planner.config.service.url")
+
+  launch.command: |
```

Comment:
@andreaturli helped me to write this. If there is a simpler way to write it he would probably know better than me. Andrea, should we change this one?